run_dir/static/js/worksets.js

```diff
@@ -215,6 +242,41 @@ function init_listjs2() {
     } );
 }
 
+// Initialize sorting and searching javascript plugin
+function init_list_closed_ws() {
+    // Setup - add a text input to each footer cell
+    $('#closed_ws_table tfoot th').each( function () {
+      var title = $('#closed_ws_table thead th').eq( $(this).index() ).text();
+      $(this).html( '<input size=10 type="text" placeholder="Search..." />' );
```

Comment:
I get two search boxes if I switch back to the regular worksets tab. It disappears when I go back to this tab though.

Reply:
Fixed by adding `$('#closed_ws_table_filter').remove();` to line 92
And `if($('#closed_ws_table_filter').length){
      $('#closed_ws_table_filter').hide(); 
    }` to line 128